data/json/mapgen/house/house_quiverfull.json

```diff
@@ -109,7 +109,7 @@
         { "group": "bed", "x": [ 20, 21 ], "y": [ 14, 15 ], "chance": 90, "repeat": [ 1, 3 ] },
         { "group": "bedroom", "x": [ 15, 21 ], "y": [ 13, 17 ], "chance": 80, "repeat": [ 1, 3 ] },
         { "item": "american_flag", "x": [ 2, 2 ], "y": [ 5, 5 ], "chance": 100 },
-        { "item": "holybook_bible1", "x": [ 2, 2 ], "y": [ 6, 6 ], "chance": 100 },
+        { "group": "holybook_bible1", "x": [ 2, 2 ], "y": [ 6, 6 ], "chance": 100 },
         { "item": "quiver_large", "x": [ 2, 2 ], "y": [ 7, 7 ], "chance": 100 }
```

Comment:
This causes an error because there is no "holybook_bible1" item_group defined.

Reply:
@jesseclvrt here's an itemgroup that accomplishes what you're trying to do:
```

  {
    "type" : "item_group",
    "id" : "house_quiverfull_bible",
    "items": [
      [ "holybook_bible1", 1 ]
    ]
  },

```
You just need to replace holybook_bible1 with the item_group id and put the item_group in house_quiverfull.json or itemgroups/locations.json